storybook/src/components/FileList/FileList.stories.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import { FileList } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+import { FileInputWithFileList } from './FileInputWithFileList'
+
+const sampleDataTransfer = new DataTransfer()
+sampleDataTransfer.items.add(new File(['sample1'], 'sample1.txt', { type: 'text/plain', lastModified: Date.now() }))
+sampleDataTransfer.items.add(new File(['sample2'], 'sample2.txt', { type: 'text/plain', lastModified: Date.now() }))
+const sampleFiles = sampleDataTransfer.files
+
+const meta = {
+  title: 'Components/Forms/File List',
+  component: FileList,
+  args: {
+    files: sampleFiles,
+  },
+  argTypes: {
+    files: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} satisfies Meta<typeof FileList>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {}
+
+export const WithInput: Story = {
+  parameters: {
+    docs: {
+      source: {
+        code: `
```

Comment:
Can we prevent duplicating the component’s source code? Would a ‘raw’ import work like we do for Markdown files? Or something else?

Reply:
Also a tryout, it did not work out loading as a raw file but I've hidden the sourceState in canvas and used the Source doc block. This might be a good and recognisable solution for all composition type of examples.